README.md

```diff
@@ -35,35 +36,36 @@ It is intended that connecting the bits together into an engine be done at the l
-* Transaction generation, manipulation, signing and ser-/deserialization including P2PKH, P2SH, multisig
-* Address generation
-* Base58check encoding
-* Native implementation of SHA256, SHA512, SHA512_HMAC, RIPEMD-160 including NIST testvectors
-* Native constant time AES (+256CBC) cipher implementation including NIST testvectors
-* Event based dogecoin P2P client capable of connecting to multiple nodes in a single thread (requires [libevent](https://github.com/libevent/libevent))
 
-## Why C? 
+- Generating and storing private and public keys
+- ECDSA secp256k1 signing and verification (through [libsecp256k1](https://github.com/bitcoin-core/secp256k1) included as git subtree)
+- Generate recoverable signatures (and recover pubkey from signatures)
+- BIP32 hierarchical deterministic key derivation
+- Transaction generation, manipulation, signing and ser-/deserialization including P2PKH, P2SH, multisig
+- Address generation
+- Base58check encoding
+- Native implementation of SHA256, SHA512, SHA512_HMAC, RIPEMD-160 including NIST testvectors
+- Native constant time AES (+256CBC) cipher implementation including NIST testvectors
+- Event based dogecoin P2P client capable of connecting to multiple nodes in a single thread (requires [libevent](https://github.com/libevent/libevent))
+
+## Why C?
 
-The Dogecoin Core project is written in C++, why move to C? This is a good question. 
+The Dogecoin Core project is written in C++, why move to C? This is a good question.
 
-The Dogecoin Core project was inherited when Dogecoin was originally forked and makes use of some reasonable heavy C++ libraries that add complexity to the build process, as well as cognitive complexity for new developers. 
+The Dogecoin Core project was inherited when Dogecoin was originally forked and makes use of some reasonable heavy C++ libraries that add complexity to the build process, as well as cognitive complexity for new developers.
 
-The desire is to provide a simple to learn library with few external dependencies that can be built with relatively little setup by new developers.  Furthermore the aim of providing wrappers for a number of higher-level languages leans strongly toward either C or RUST from a binding/support perspective, and we believe C still has significantly more support when writing bindings for a wide variety of other languages.
+The desire is to provide a simple to learn library with few external dependencies that can be built with relatively little setup by new developers.  Furthermore the aim of providing wrappers for a number of higher-level languages leans strongly toward either C or RUST from a binding/support perspective, and we believe C still has significantly more support when writing bindings for a wide variety of other languages. Wrappers that previously lived here in this repository have been moved to their own respective projects (python and go bindings) found at [@dogeorg](https://github.com/dogeorg).
```

Comment:
Something something C better supported on embedded/lightweight platforms, with the goal of moving dogecoin beyond the realm of the PC; and actually prepares (constrains it) to be re-coded/adapted in other languages as specific OO and memory mgmt functions of C++ aren't required.  Or something.